### PR TITLE
Adding Oxygen level config option

### PR DESCRIPTION
**Pull request by alberttheprince**
## Description

Added a config option to change the oxygen level via the config so it can be set without editing the main client file.

## Checklist

- [x] I have personally loaded this code into an updated Qbox project and checked all of its functionality.
- [x] My code fits the style guidelines.
- [x] My PR fits the contribution guidelines.

### CODE REVIEW

**Comment by DevX32** [comment]
add -- in seconds in config

**Comment by alberttheprince** [comment]
> add -- in seconds in config

done, and replaced the other instance where oxygen level ~= 100

File: client/main.lua [hunk 1]
```diff
@@ -350,7 +350,7 @@ RegisterNetEvent('qb-diving:client:UseGear', function()
             CreateThread(function()
                 while currentGear.enabled and IsPedSwimmingUnderWater(cache.ped) do
```
Comment (TonybynMp4) [inline client/main.lua]:
```suggestion
                while currentGear.enabled and oxygenLevel > 0 do
                    oxygenLevel -= 1
                    if oxygenLevel % 10 == 0 and oxygenLevel ~= Config.OxygenLevel then
                        TriggerServerEvent("InteractSound_SV:PlayOnSource", "breathdivingsuit", 0.25)
                    end
                    if oxygenLevel == 0 then
```

**Review by TonybynMp4 — CHANGES_REQUESTED** [review]

File: client/main.lua (hunk 1 above)
Comment (TonybynMp4) [reply client/main.lua]: 0%10 == 0 so the elseif is never ran

**Review by TonybynMp4 — APPROVED** [review]
once it's merged with the previous prs it'll be good👍🏻

**Review by Manason — APPROVED** [review]

**Pull request merged**